content/docs/introduction/faq.md

```diff
@@ -199,15 +199,15 @@ you an idea, here are some results from benchmarks:
 
 * On an older 8-core machine with Intel Core i7 CPUs and two spinning
   disks (Samsung HD753LJ) in a RAID-1 setup, Prometheus sustained an
-  ingestion rate of 20k samples per second, belonging to 450k time
-  series, scraped from 1500 targets.
+  ingestion rate of 18k samples per second, belonging to 56k time
```

Comment:
Unfortunately no. With more time series (and thus more time series files), you hit a cliff where all of a sudden, you cannot write more than 100 chunks a sec anymore, but more something like 50 (or, if there is other load, only 30). Perhaps the file system knows how to optimize smaller amount of files or data, or, more likely, the write cache spans the writes to 56k files nicely, but not 560k. With my latest tweaks, everything still works with 560k, because Prometheus is doing the queuing itself (if you have enough memory chunks configured, say 3x number of time series), but then you have millions of chunks in the persist queue. If you want to shutdown the server, those millions of chunks have to be persisted, which will take a long time (O(hours)). So it's not a scenario you would recommend to users right now. See https://github.com/prometheus/prometheus/issues/533 for the details how to remedy the situation.